requirements/run.txt

```diff
@@ -81,7 +89,7 @@ python-daemon==2.3.1
     # via -r requirements/run.in
 python-engineio==4.3.4
     # via python-socketio
-python-openflow
+python-openflow==2021.1
```

Comment:
We need to leave `python-openflow` unpinned until this issue https://github.com/kytos-ng/kytos/issues/271 is prioritized in this future, otherwise it'd pull a wrong and old version.

Reply:
Changed.